front/src/views/sign-up/sign-up.view.js

```diff
@@ -23,12 +24,14 @@ export default {
   methods: {
     async signUp () {
       if (this.valid) {
```

Comment:
Nit: you can replace the check with the variable this.valid for disabling the "continue" button: <v-btn ...: disable=!valid... />

Reply:
ow, but it wasn’t on the design, so I didn’t do it C: